data/human/culture conversations.txt

```diff
@@ -1101,3 +1101,279 @@ mission "Little Vienna in New Austria"
 			label backtoship
 			`	With the atmosphere steeped in nostalgia for a distant city on a distant world long past, you return to your ship.`
 
+phrase "racetrack"
+	word
+		"Racetrack"
+		"Speedway"
+		"Arena"
+		"Course"
+		"Circuit"
+		"Glideway"
+		"Hover Circuit"
+		"Velocity Course"
+		"Hoverloop"
+
+phrase "sunracer opening"
+	word
+		"As you walk through the din of Sunracer's bustling spaceport"
+		"As you wade through the clang and clatter of Sunracer's ever-churning shipyards"
+		"Amidst the city's persisting glitz"
+		"Lost in a maze of hangars and half-built hulls"
+		"Navigating through the organized chaos of Megaparsec shipyards"
+
+phrase "sunracer hovercraft adhook"
+	word
+		"dazzling holographic display"
+		"radiant, floating billboard"
+		"vivid holographic advertisement"
+	word
+		" seizes your attention"
+		" catches your eye"
+		" appears overheard"
+
+phrase "sunracer hovercraft adcloser"
+	word
+		"The ad flashes sleek"
+		"Photo-realistic images show agile"
+		"The visuals display streamlined"
+	word
+		" hovercraft "
+		" vehicles "
+	word
+		"slicing through a twisting course in untamed terrain, "
+		"racing along a serpentine, rugged track, "
+		"carving through wild, winding roads, "
+	word
+		"with the ad copy "
+		"the display's text "
+		"the words below "
+	word
+		"promising high-speed thrills for the audience"
+		"teasing a spectacle of breakneck action"
```

Comment:
Again, I don't think most people would use "teasing" in this context. It sort of works, but it's a bit awkward. Maybe:

```suggestion
		"providing a teaser of the breakneck action to come"
```